src/Alpacka.Lib/Instances/MultiMC/MultiMCHandler.cs

```diff
@@ -48,11 +53,13 @@ public void Install(string instancePath, ModpackBuild pack)
         public void Update(string instancePath, ModpackBuild oldPack, ModpackBuild newPack)
         {
             var multiMCInstancePath = Path.GetDirectoryName(instancePath);
+            var multiMCInstanceCfg = Path.Combine(multiMCInstancePath, CONFIG_FILE);
             
             // update minecraft and forge version
-            if (newPack.MinecraftVersion != oldPack?.MinecraftVersion)
-                MultiMCInstance.UpdateVersion(multiMCInstancePath, newPack.MinecraftVersion, newPack.ForgeVersion);
-            
+            var cfg = new MultiMCConfig(multiMCInstanceCfg);
```

Comment:
Why remove the if statement? I think I meant to use `ForgeVersion` instead of `MinecraftVersion`, but it seems odd to modify the config if neither of them have changed. (I assume changing MC version automatically means the Forge version would change, too.)